config/initializers/omniauth.rb

```diff
@@ -13,10 +13,10 @@ def select_field(hash)
 
       label_field(key, key.capitalize)
 
-      @html << "\n<select name='#{key}'>\n"
+      @html << "\n<select style='padding: 5px; margin: 5px auto 20px;' name='#{key}'>\n"
 
       values.each do |value|
-        @html << "\n<option value='#{value}'>#{value}</option>"
+        @html << "\n<option style='padding: 10px auto' value='#{value}'>#{value}</option>"
```

Comment:
J'veux bien que tu m'expliques pourquoi on fait du html dans ce fichier. J'ai pas tout compris du commentaire :
(j'ai pas de reco d'actions à effectuer je veux juste comprendre)

> NOTE: this code allows for multiple-choice fields in the Omniauth
> dev strategy, rendered as <select> elements. It's not the best code
> but it will allow the team to try logging in as any establishment
> with the relevant ministry.

Reply:
c'est juste le formulaire par défaut de Omniauth, que j'avais modifié pour permettre des `select` au lieu de simple `input type=text`